### PR TITLE
Cherry pick doc changes from 2.3

### PR DESCRIPTION
**Pull request by dedemorton**
Contains doc changes from 2.3 that are not currently in 2.x.

### CODE REVIEW

File: docs/static/logstash-docs-home.asciidoc [hunk 1]
```diff
@@ -1,30 +0,0 @@
-[[logstash-docs-home]]
```
Comment (andrewvc) [inline docs/static/logstash-docs-home.asciidoc]: Why was this removed?

**Comment by andrewvc** [comment]
@dedemorton LGTM aside from the one deletion. Why was that page removed?

**Comment by dedemorton** [comment]
@andrewvc The page that I deleted is no longer being used anywhere in the docs. Suyog said it was OK to remove it. Is there some reason why it should not be removed?

**Comment by dedemorton** [comment]
I'm going to merge this now because the doc build really happens in logstash-docs anyhow, where the file has already been removed. :-)
